test/integration/change_edition_type_test.rb

```diff
@@ -1,261 +1,93 @@
-    assert page.has_content?("New edition created")
-    assert page.has_content?(guide.whole_body)
-  end
-
-  test "should be able to convert a LicenceEdition into an AnswerEdition" do
-    licence = FactoryGirl.create(:licence_edition, state: 'published', licence_overview: "Licence overview content", licence_short_description: "Short description content")
-    visit_edition licence
-
-    within "div.tabbable" do
-      click_on "Admin"
-    end
-
-    assert page.has_button?("Create as new Answer edition")
-
-    click_on "Create as new Answer edition"
-
-    assert page.has_content?(licence.title)
-    assert page.has_content?("New edition created")
-
-    assert_field_contains("Licence overview content", "Body")
-    assert_field_contains("Short description content", "Body")
-  end
-
-  test "should be able to convert a ProgrammeEdition into an AnswerEdition" do
-    programme = FactoryGirl.create(:programme_edition, state: 'published')
-    visit_edition programme
-
-    within "div.tabbable" do
-      click_on "Admin"
-    end
-
-    assert page.has_button?("Create as new Answer edition")
-
-    click_on "Create as new Answer edition"
-
-    assert page.has_content?(programme.title)
-    assert page.has_content?("New edition created")
-
-    assert_field_contains("Overview", "Body")
-    assert_field_contains("What you'll get", "Body")
-    assert_field_contains("Eligibility", "Body")
-    assert_field_contains("How to claim", "Body")
-    assert_field_contains("Further information", "Body")
-  end
-
-  test "should be able to convert a TransactionEdition into an AnswerEdition" do
-    transaction = FactoryGirl.create(:transaction_edition, slug: "childcare", state: 'published')
-    visit_edition transaction
-
-    within "div.tabbable" do
-      click_on "Admin"
-    end
-
-    assert page.has_button?("Create as new Answer edition")
-
-    click_on "Create as new Answer edition"
-
-    assert page.has_content?(transaction.title)
-    assert page.has_content?("New edition created")
-    assert_field_contains(transaction.more_information, "Body")
-    assert_field_contains(transaction.alternate_methods, "Body")
-  end
-
-  test "should be able to convert a SimpleSmartAnswerEdition into an AnswerEdition" do
-    simple_smart_answer = FactoryGirl.create(:simple_smart_answer_edition, state: 'published')
-    visit_edition simple_smart_answer
-
-    within "div.tabbable" do
-      click_on "Admin"
-    end
-
-    assert page.has_button?("Create as new Answer edition")
-
-    click_on "Create as new Answer edition"
-
-    assert_text(simple_smart_answer.title)
-    assert_text("New edition created")
-  end
-
-# tests for changing Answer, Guide, Programme into a Transaction
+  edition_types = [:transaction_edition, :place_edition, :programme_edition, :simple_smart_answer_edition, :business_support_edition, :answer_edition, :guide_edition, :licence_edition, :video_edition, :completed_transaction_edition, :help_page_edition, :campaign_edition]
```

Comment:
Makes sense to me, means when a new format is created the conversion tests will update automatically. Paul is on a beach somewhere therefore he agrees. Updated it.